src/App.tsx

```diff
@@ -33,14 +35,26 @@ export default class App extends React.Component {
             Coveo Headless + Material UI
           </Typography>
         </Box>
-
+        <div className="tabBar-container">
+          <Tab
+            initialState={anyProps.initialState!}
+            options={anyProps.options!}
+          />
+          <Tab
+            initialState={intelProps.initialState!}
+            options={intelProps.options!}
+          />
+          <Tab
+            initialState={AMD_Props.initialState!}
```

Comment:
It's not the best practice to mix the snake_case with camelCase, so I'm advocating for camelCasing of `AMD_Props` 😉
for better readability, we can right `amd` instead of `AMD`.

Let me know what you think. If you think this suggestion is worth to implement, we'd need to updated the naming throughout the PR 

```suggestion
            initialState={amdProps.initialState!}
```

Reply:
will do!

Reply:
Hey @eguyon-coveo , I see your updates, thx a lot for adding them 👍
Still, I see a snake_case naming in this PR 🥲

Reply:
you're right...